arm/cores/Arduino.h

```diff
@@ -85,6 +86,17 @@ extern "C" {
     /*
     * XMC pin interrupt type
     */
+#if defined (XMC4700_Radar_Baseboard)
```

Comment:
Conditional compiling for each of the board variants at this level should be avoided. Core components implementation should be consistant and available for all microcontroller families, as long as they have an ERU hardware peripheral. 
It should be generic for all hardware platforms...

Reply:
You have that the WRONG way round the core functions should as MUCH as possible be board agnostic and rely on flags for support type like CCU_4 and CCU_8, not board specific (platform) parameters these should be dealt with at LOWER levels

You are making assumptions about your board is the only one being used and the long term support issues it will cause see comments in #95 on a similar vein